src/parser/tests.rs

```diff
@@ -583,8 +615,9 @@ pub fn testing_global_def() {
     setup_logger();
     let mut err = ErrorGen::new("".to_string(), "".to_string(), 0);
     let script = r#"
+        (i32, i32) sample = (1, 2);
         dummy_fn() {
-            a = strcmp((arg0, arg1), "bookings");
+            a = strcmp(sample, "bookings");
             strcmp((arg0, arg1), "bookings");
         }
         i32 i;
```

Comment:
Add as new test instead, more tests is better.

Reply:
added to the list of "valid scripts"